Blog/include/functions.php

```diff
@@ -44,16 +44,38 @@ function getComments($blogPostID){ //colon is a security prevention
 		WHERE blogPostID = :blogPostID
 	",
 	array('blogPostID' => $blogPostID)
-);
+)->fetchAll();
 
-	return $result->fetchAll();
+	return $result;
 
 }
 
 function printHeader(){
 		echo "<html>";
 };
 
+@$userID = $_REQUEST['userID'];
+@$Comment = $_REQUEST['Comment'];
+
+//var_dump($_REQUEST);
 
+function insertBlogComment($blogPostID, $userID, $commentTimeStamp, $Comment){
+	if(isset($_REQUEST['submitComment'])){
+		$result = dbQuery("
+		INSERT INTO Comments(blogPostID, userID, commentTimeStamp, Comment)
+		VALUES ('$blogPostID','$userID', '$commentTimeStamp', '$Comment')
+		")->fetch();
+	}
+}
+
+function submitComment($userID, $commentTimeStamp, $Comment){
```

Comment:
I'd come up with a naming scheme for any function that echos html rather than running a query. When I see `submitComment` I think this is going to save a comment to the db. Maybe call it `echoSubmitCommentConfirmation` or something like that.

(btw, it's not totally clear to me what this function is used for)